src/frontend/components/property-type/datetime/filter.tsx

```diff
@@ -24,13 +24,13 @@ const Filter: React.FC<FilterPropertyProps> = (props) => {
         <Label>{`- ${translateProperty('from')}: `}</Label>
         <DatePicker
           value={fromValue}
-          onChange={(date: Date): void => onChange(fromKey, date)}
+          onChange={(date): void => onChange(fromKey, date)}
```

Comment:
By the way, I think it'd be a great DX improvement if we removed that ESLint rule forcing us to add return types to every function. Typescript has type inference for a reason and typing function return types is only recommended on functions that need to expose a very specific API. An anonymous `onChange` handler doesn't _need_ to return `void`.

Reply:
Good point. Added rule to eslint and removed redundant return types.